src/transformers/models/conditional_detr/modeling_conditional_detr.py

```diff
@@ -86,6 +86,8 @@ class ConditionalDetrDecoderOutput(BaseModelOutputWithCrossAttentions):
         intermediate_hidden_states (`torch.FloatTensor` of shape `(config.decoder_layers, batch_size, num_queries, hidden_size)`, *optional*, returned when `config.auxiliary_loss=True`):
             Intermediate decoder activations, i.e. the output of each decoder layer, each of them gone through a
             layernorm.
+        reference_points (`torch.FloatTensor` of shape `(config.decoder_layers, batch_size, num_queries, 4 (anchor points))`):
+            Reference points (reference points of each layer of the decoder).
```

Comment:
I fixed in ConditionalDetr, added "2 (anchor points)->4 (anchor points)". The run 'make repo-consistency' messed up everything. What am I missing to prevent this from happening?:)

Reply:
Hmm, maybe this cannot be handled by copied from

Reply:
I'll modify it!